specification/mediaservices/resource-manager/Microsoft.Media/stable/2022-08-01/examples/async-operation-result.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "subscriptionId": "0a6ec948-5a62-437d-b9df-934dc7c1b722",
+    "resourceGroupName": "mediaresources",
+    "accountName": "slitestmedia10",
+    "operationId": "62e4d893-d233-4005-988e-a428d9f77076",
+    "api-version": "2022-08-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "error": {
+          "code": "None",
+          "message": null,
+          "target": "d7aea790-8acb-40b9-8f9f-21cc37c9e519"
+        },
```

Comment:
why will 200 have error in the body? remove?

Reply:
Yes, 200 will still have the `error` property. This is just to get the status of the long running operation. So 200 just means it gets the operation status successfully, but the status itself could be Succeeded or Failed or InProgress. Customers can ignore the `error` field if the operation is not failed.

Reply:
sounds good.